common/persistence/cassandra/cassandraPersistence.go

```diff
@@ -63,6 +63,9 @@ const (
 	rowTypeReplicationDomainID   = "10000000-5000-f000-f000-000000000000"
 	rowTypeReplicationWorkflowID = "20000000-5000-f000-f000-000000000000"
 	rowTypeReplicationRunID      = "30000000-5000-f000-f000-000000000000"
+	// Row Constants for Replication Task DLQ Row. Source cluster name will be used as WorkflowID.
+	rowTypeDLQDomainID = "10000000-6000-f000-f000-000000000000"
+	rowTypeDLQRunID    = "30000000-6000-f000-f000-000000000000"
```

Comment:
```go
rowTypeDLQDomainID = "10000000-6000-f000-f000-000000000000"
rowTypeDLQWorkflowID = "10000000-6000-f000-f000-000000000000"
rowTypeDLQRunID    = "30000000-6000-f000-f000-000000000000"
```
and put the source cluster name in the task or blob

Reply:
Discussed offline. We will have source cluster in the DLQ because source task ID can have collision. On the other side cluster name string is error prone, so we will use initial failover version.